packages/core/src/browser/style/variables-bright.useable.css

```diff
@@ -197,7 +197,7 @@ is not optimized for dense, information rich UIs.
   --theia-scrollbar-active-rail-color: transparent;
 
   /* Menu */
-  --theia-menu-color0: var(--theia-layout-color2);
+  --theia-menu-color0: var(--theia-layout-color3);
```

Comment:
[minor]: Strange that both `--theia-menu-color0:` and `--theia-menu-color2:` map to the same layout color. Same for the dark theme.

Reply:
In dark theme --theia-menu-color0 and --theia-menu-color2 are different.